WordPress/Sniffs/WhiteSpace/CastStructureSpacingSniff.php

```diff
@@ -15,21 +19,13 @@
  * @package WPCS\WordPressCodingStandards
  *
  * @since   0.3.0
+ * @since   0.xx.0 This sniff now extends the Squiz_Sniffs_WhiteSpace_CastSpacingSniff class it was based upon.
+ * @since   0.xx.0 This sniff now has the ability to fix the issues it flags.
```

Comment:
Are you planning to fill out the version numbers just before merge, or at the time of release? (Just so that we're all on the same page.)